src/PSRClassesInterface.jl

```diff
@@ -7,6 +7,15 @@ import JSON
     error("Julia version $VERSION not supported by PSRClassesInterface, upgrade to 1.6 or later")
 end
 
+const PSRCLASSES_DEFAULT = Dict{String,Any}()
+
+function __init__()
+    merge!(
+        PSRCLASSES_DEFAULT,
+        JSON.parsefile(joinpath(@__DIR__, "json_metadata", "psrclasses.default.json"))
+    )
+end
```

Comment:
Don't use `__init__`. It is bad for compiling the code later.

Do something similar from the pmd files.
Have that path as default kwag in the create/init study function, but allow the user tho pass their own files.